content/millennium/dstu2/scheduling/slot.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Slot | DSTU 2 API
+---
+
+# Slot
+
+* TOC
+{:toc}
+
+## Overview
+
+Slot resources are used to provide time-slots that can be booked using an appointment. They do not provide any information about appointments that are available, just the time, and optionally what the time can be used for.
+<br/><br/>
+The following fields are returned if valued:
+
+* [Type](http://hl7.org/fhir/DSTU2/slot-definitions.html#Slot.type){:target="_blank"}
+* [Schedule](http://hl7.org/fhir/DSTU2/slot-definitions.html#Slot.schedule){:target="_blank"}
+* [Start](http://hl7.org/fhir/DSTU2/slot-definitions.html#Slot.start){:target="_blank"}
+* [End](http://hl7.org/fhir/DSTU2/slot-definitions.html#Slot.end){:target="_blank"}
+* [Free-Busy Type](http://hl7.org/fhir/DSTU2/slot-definitions.html#Slot.freeBusyType){:target="_blank"}
+* [Location Extension](#extensions)
+
+## Terminology Bindings
+
+<%= terminology_table(:schedule, :dstu2) %>
+
+## Extensions
+
+* [Location]
+
+### Custom Extensions
+
+All URLs for custom extensions are defined as `https://fhir-ehr.cerner.com/dstu2/StructureDefinition/{id}`
+
+ID             | Value\[x] Type    | Description
+---------------|-------------------|----------------------------------------------------------------------------------
+`location`      | [`Reference`]          | Reference to the location corresponding to this schedule, where the appointment can be booked.
+
+## Search
+
+Search for Schedules that meet supplied query parameters:
+
+    GET /Slot?:parameters
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, patient: true, system: true) %>
+
+### Parameters
+
+ Name             | Required?                                                  | Type           | Description
+------------------|------------------------------------------------------------|----------------|------------------------------------------------------------------------------------
+`_id`             | This, or `type`                                            | [`token`]      | The logical resource id associated with the resource.
+`type`            | Yes when using `schedule.actor` and\or `location`, or `id` | [`token`]      | The type of appointments that can be booked into this slot. Example: `12345`
+`schedule.actor`  | This and `type`, or `id`                                   | [`reference`]  | A single or comma separated list of Practitioner references. Example: `12345`
```

Comment:
Do we want to note that this is a chained parameter?

Also, that example seems surprising, since the actor reference can be many types. I assume this example should be 'Practitioner/12345'?

Do we allow actors other than practitioners on the Schedule?